database/seed.js

```diff
@@ -32,15 +33,13 @@ for (var j = 0; j < 200; j++) {
   sampleAnswerData.push(answers);
 }
 
-let copiedAns = sampleAnswerData.slice();
 
 for (var i = 0; i < 100; i++) {
+  let numOfAnswers = Math.floor((Math.random() * 3))
   let answered = [];
   for (var k = 0; k < numOfAnswers; k++) {
-    answered.push(copiedAns.pop());
+    answered.push(sampleAnswerData.pop());
   }
-
-
   var randomQuestion = faker.hacker.phrase();
   function Question(body) {
     this.body = randomQuestion;
```

Comment:
could just concat a questionmark here if you want it to look more like a question!